src/SFA.DAS.EmployerFinance/Configuration/GoogleAnalyticsConfigurationFactory.cs

```diff
@@ -0,0 +1,32 @@
+using SFA.DAS.AutoConfiguration;
+
+namespace SFA.DAS.EmployerFinance.Configuration
+{
+    public class GoogleAnalyticsConfigurationFactory : IGoogleAnalyticsConfigurationFactory
+    {
+        private readonly IEnvironmentService _environmentService;
+
+        public GoogleAnalyticsConfigurationFactory(IEnvironmentService environmentService)
+        {
+            _environmentService = environmentService;
+        }
+
+        public GoogleAnalyticsConfiguration CreateConfiguration()
+        {
+            var configuration = new GoogleAnalyticsConfiguration();
+
+            if (_environmentService.IsCurrent(DasEnv.PREPROD))
+            {
+                configuration.ContainerId = "GTM-KWQBWGJ";
```

Comment:
Can you confirm with Pritesh or Ali that these are the correct ContainerId & TrackingId values for Employer Finance?

Reply:
I'd already asked Ali for them so they are the correct values